src/components/network/network-panel.html

```diff
@@ -4,6 +4,13 @@
     <button class="btn btn-default glyphicon glyphicon-refresh btn-update"></button>
     <input class="form-control input-with-update">
   </div>
+  <div id="input-type" class="panel-section">
+    <div class="item-title">Input Type</div>
+      <div class="input-checkbox">
+        <label id="regex" class="radio-inline"><input type="radio" name="optradio">regex</label>
```

Comment:
Please avoid sub-element level ids. Move id to the parent container. Select parent container by id and child elements by other attributes such as aria-roles.